lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -283,15 +283,19 @@ protected function getPortableNamespaceDefinition(array $namespace)
      */
     protected function _getPortableSequenceDefinition($sequence)
     {
-        if ($sequence['schemaname'] != 'public') {
+        if ($sequence['schemaname'] !== 'public') {
             $sequenceName = $sequence['schemaname'] . "." . $sequence['relname'];
         } else {
             $sequenceName = $sequence['relname'];
         }
 
-        $data = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
+        if ( ! isset($sequence['increment_by'], $sequence['min_value'])) {
+            $data                     = $this->_conn->fetchAll('SELECT min_value, increment_by FROM ' . $this->_platform->quoteIdentifier($sequenceName));
```

Comment:
You can use `Connection::fetchAssoc()` to fetch one row and then `array_merge($sequence, $data)` to not confuse the keys.

Reply:
That would require rework of other tests https://travis-ci.org/doctrine/dbal/jobs/320775111 However, when I launch the testsuite locally, they all pass.

Reply:
Which exactly failures do you think are caused by this?

Reply:
My bad, found it, fixed it